be/src/runtime/runtime_state.cpp

```diff
@@ -101,18 +101,22 @@ RuntimeState::RuntimeState(const TQueryGlobals& query_globals)
     _query_options.batch_size = DEFAULT_CHUNK_SIZE;
     if (query_globals.__isset.time_zone) {
         _timezone = query_globals.time_zone;
-        _timestamp_ms = query_globals.timestamp_ms;
+        if (query_globals.__isset.timestamp_us) {
+            _timestamp_us = query_globals.timestamp_us;
+        } else {
+            _timestamp_us = query_globals.timestamp_ms * 1000;
+        }
     } else if (!query_globals.now_string.empty()) {
         _timezone = TimezoneUtils::default_time_zone;
         DateTimeValue dt;
         dt.from_date_str(query_globals.now_string.c_str(), query_globals.now_string.size());
         int64_t timestamp;
         dt.unix_timestamp(&timestamp, _timezone);
-        _timestamp_ms = timestamp * 1000;
+        _timestamp_us = timestamp * 1000000;
```

Comment:
what's the unit of timestamp? ms?   ms=1000*us

Reply:
> what's the unit of timestamp? second?

yes.